src/seedu/addressbook/ui/TextUi.java

```diff
@@ -164,5 +168,16 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
-
+	
+	/**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+	public String getPrintableString(Printable... printables) {
+     	final StringBuilder builder = new StringBuilder();
+     	for (Printable p : printables) {
+     		builder.append(p.getPrintableString() + PRINTABLE_SEPARATOR);
+     	}
+     	builder.replaceAll(REMOVE_TRAILING_COMMA, REPLACE_TRAILING_COMMA);
```

Comment:
Maybe a StringJoiner will be more efficient and you wouldn't have to handle the trailing comma like this.